frontend/bindings_from_node.ts

```diff
@@ -0,0 +1,276 @@
+// This file has been generated by Specta. DO NOT EDIT.
```

Comment:
This is the result of all the types that Specta generated from Graphite's codebase.

These can be used anywhere in Graphite's frontend.

Reply:
Shouldn't this be gitignored?

Reply:
And is there a way to get Specta to generate `undefined` instead of `null` in this file? We avoid `null` by convention.

Reply:
> Shouldn't this be gitignored?

I would generally advise against this although feel free to do whatever you think is best. I personally think having the types available in code review and when you clone in the codebase is important.

> And is there a way to get Specta to generate undefined instead of null in this file? We avoid null by convention.

I think you can use `#[specta(optional)]` on each of the fields.